client/lib/plans/plans-list.js

```diff
@@ -822,20 +822,37 @@ export const PLANS_LIST = {
 		getAudience: () => i18n.translate( 'Best for students' ),
 		getProductId: () => 2002,
 		getStoreSlug: () => constants.PLAN_JETPACK_FREE,
-		getTagline: ( feature ) => {
-			switch ( feature ) {
-				case constants.FEATURE_JETPACK_BACKUP_DAILY:
-				case constants.FEATURE_JETPACK_BACKUP_DAILY_MONTHLY:
-				case constants.FEATURE_JETPACK_BACKUP_REALTIME:
-				case constants.FEATURE_JETPACK_BACKUP_REALTIME_MONTHLY:
-					return i18n.translate(
-						'Upgrade your site to access additional features, including spam protection and priority support.'
-					);
-				default:
-					return i18n.translate(
-						'Upgrade your site to access additional features, including spam protection, backups, and priority support.'
-					);
+		getTagline: ( siteFeatures = [] ) => {
+			const hasSiteJetpackBackup = siteFeatures.some( ( feature ) =>
+				[
+					constants.FEATURE_JETPACK_BACKUP_DAILY,
+					constants.FEATURE_JETPACK_BACKUP_DAILY_MONTHLY,
+					constants.FEATURE_JETPACK_BACKUP_REALTIME,
+					constants.FEATURE_JETPACK_BACKUP_REALTIME_MONTHLY,
+				].includes( feature )
+			);
+			const hasSiteJetpackScan = siteFeatures.some( ( feature ) =>
+				[
+					constants.FEATURE_JETPACK_SCAN_DAILY,
+					constants.FEATURE_JETPACK_SCAN_DAILY_MONTHLY,
+				].includes( feature )
+			);
+			if ( hasSiteJetpackBackup && hasSiteJetpackScan ) {
+				return i18n.translate(
+					'Upgrade your site to access additional features, including spam protection and priority support.'
+				);
+			} else if ( hasSiteJetpackBackup ) {
+				return i18n.translate(
+					'Upgrade your site to access additional features, including spam protection, security scanning, and priority support.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Upgrade your site to access additional features, including spam protection, real-time backups, and priority support.' )` *ES Score: 7*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/context-driven-plans-products-copy)